services/venice-controller/src/main/java/com/linkedin/venice/controller/stats/TopicCleanupServiceStats.java

```diff
@@ -0,0 +1,33 @@
+package com.linkedin.venice.controller.stats;
+
+import com.linkedin.venice.stats.AbstractVeniceStats;
+import com.linkedin.venice.stats.Gauge;
+import io.tehuti.metrics.MetricsRepository;
+import io.tehuti.metrics.Sensor;
+import io.tehuti.metrics.stats.Count;
+
+
+public class TopicCleanupServiceStats extends AbstractVeniceStats {
+  private final Sensor deletableTopicsCountGaugeSensor;
+  private final Sensor deletedTopicsCountSensor;
+  private final Sensor topicDeletionErrorCountSensor;
+
+  public TopicCleanupServiceStats(MetricsRepository metricsRepository) {
+    super(metricsRepository, "TopicCleanupService");
+    deletableTopicsCountGaugeSensor = registerSensorIfAbsent("deletable_topics_count", new Gauge());
+    deletedTopicsCountSensor = registerSensorIfAbsent("deleted_topics_count", new Count());
+    topicDeletionErrorCountSensor = registerSensorIfAbsent("topic_deletion_error_count", new Count());
```

Comment:
Shall we use `Rate` instead of `Count`?

Reply:
Good point, we don’t really care the exact count as much as the overall rate. Let me update it.